src/api.rs

```diff
@@ -63,23 +64,27 @@ pub fn load_string(input: String) -> Result<Engine, Error> {
     ExecutionEngine::initialize(program)
 }
 
-/// Executes the reasoning process of the [`Engine`] and returns a vector of result [`predicates`][Predicate].
+/// Executes the reasoning process of the [`Engine`].
 ///
 /// # Note
 /// If there have been `@source` routines in the parsed rules, all relative paths are resolved with the current working directory
```

Comment:
Could you please replace `have been` with `are` (again, this was broken before, but please don't leave it this way)?